Makefile

```diff
@@ -215,7 +215,7 @@ TMP_DIR=$$(mktemp -d) ;\
 cd $$TMP_DIR ;\
 go mod init tmp ;\
 echo "Downloading $(2)" ;\
-GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+GOBIN=$(PROJECT_DIR)/bin go install $(2) ;\
```

Comment:
I think all the tmpdir and go mod init business above also becomes unneeded and should be dropped(?)

Reply:
done